### PR TITLE
Update LICENSE to mit with riders

### PR DESCRIPTION
**Pull request by faddat**
## 1. Summary

Makes governance the only licensor of the code.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,201 +1,9 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Copyright <YEAR> <COPYRIGHT HOLDER>
```
Comment (joe-bowman) [inline LICENSE:1]: Does this need completing?

Comment (faddat) [reply LICENSE:1]: sure does

**Comment by faddat** [comment]
PS: does this look right to you?  I am not 100% sure nor am I a lawyer.  But I love the idea of gov controlling the use of the chain code.

**Comment by joe-bowman** [comment]
Do we need to ask governance to change this license in the first instance?

**Comment by faddat** [comment]
I guess I don't think so?  

As the contributors / originaltors of the original license, I think we could just change it, HOWEVER!

I think we should ask governance.  :)

**Review by joe-bowman — APPROVED** [review]
Ship it

**Pull request merged**